pronto/api/signatures/proteins.py

```diff
@@ -299,7 +358,7 @@ def get_proteins_alt(accessions):
             "comment": comment_value,
             "description": name_value,
             "exclude": list(exclude),
-            "go": f"{term_id}: {term_name}" if term_name else None,
+            "go": f"{go_id}: {term_name}" if term_name else None,
```

Comment:
Again, we want to have a clear separation between filters, so we return an extra `violate-go` property:

```suggestion
            "go": f"{term_id}: {term_name}" if term_name else None,
            "violate-go": (f"{violate_term_id}: {violate_term_name}"
                           if violate_term_name else None),
```